packages/create-redwood-app/template/redwood.toml

```diff
@@ -14,5 +14,3 @@
   port = 8911
 [browser]
   open = true
-[experimental]
```

Comment:
Note: This option was removed from Web build although the option still exists in config. Removing it here and I'll create an issue to fully remove and update docs in v0.38